R/km.per.gridunit.R

```diff
@@ -0,0 +1,54 @@
+#' Input Speed Parameter Guidance by Martin C. Arostegui aka Dr. Mola
+#' km.per.gridunit() calculates the min and max longitudinal width (in km) of a single grid unit
```

Comment:
there's actually an @author tag, for example see https://github.com/camrinbraun/HMMoce/blob/dev/R/gausskern.R